gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/TestUtility.java

```diff
@@ -1,19 +1,23 @@
 package uk.nhs.adaptors.pss.translator;
 
+import org.hl7.v3.CD;
 import org.hl7.v3.CV;
+import org.hl7.v3.RCMRMT030101UKComponent3;
 import org.hl7.v3.RCMRMT030101UKEhrComposition;
 import org.hl7.v3.RCMRMT030101UKEhrExtract;
+import org.hl7.v3.RCMRMT030101UKEhrFolder;
 
+import java.util.List;
 import java.util.function.Function;
 
 public final class TestUtility {
     private TestUtility() { }
 
     public static final Function<RCMRMT030101UKEhrExtract, RCMRMT030101UKEhrComposition> GET_EHR_COMPOSITION =
         extract -> extract
-            .getComponent().get(0)
+            .getComponent().getFirst()
             .getEhrFolder()
-            .getComponent().get(0)
+            .getComponent().getFirst()
             .getEhrComposition();
```

Comment:
As I've learned more since joining NIA, this function is not great. This assumes that there is one `EHR Folder`, but no, this is not the case. According to the Message Implementation Manual, and `UK05` XSD schema - one `EHR Extract` can have many components, which can contain one and only one `EHR Folder`.

So this can definitely be improved.

Technically out-of-scope for this ticket though, to raise in a separate PR.

Reply:
**TODO** another PR.